### PR TITLE
[StructuralMechanics] global mass matrix selection

### LINKED ISSUE #3728: [StructuralMechanics] MassMatrix-default: lumped or consistent?
The title says it all, we should agree on how to specify that the lumped-/consistent mass-matrix is used.

Afaik currently in the Beams the lumped is default, while in the Solids the consistent one is default.
In the Shells it is not consistent (haven't had the time yet to clean it)

Also we should agree on a variable (`bool`) that is used for specifying it.
Currently used in Solids: `COMPUTE_LUMPED_MASS_MATRIX`
Currently used in Beams: `USE_CONSISTENT_MASS_MATRIX`

What do you think regarding those two things?
- Consistent vs lumped (could be different for different elements?)
- Name of variable used to specify it? Personally I like sth with `USE_XXX_MASS_MATRIX`

### PR DESCRIPTION
**Pull request by philbucher**
as discussed in #3728 
closes #3728 

After this PR the elements will be refactored to use this

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]
OK, maybe we should add the computation of the whole lumped matrix, in  theory is the same for all kind of elements

**Comment by philbucher** [comment]
Yes, I will do that and also add many other things that are duplicated across the elements

Step by step :)

**Comment by philbucher** [comment]
I had to fix a test, can you reapprove pls? :)

**Review by loumalouomega — APPROVED** [review]

**Comment by philbucher** [comment]
thx

File: applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// External includes
+
+// Project includes
+#include "structural_mechanics_element_utilities.h"
+#include "includes/variables.h"
+
+namespace Kratos {
+namespace StructuralMechanicsElementUtilities {
+
+bool ComputeLumpedMassMatrix(
```
Comment (RiccardoRossi) [inline applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp:24]: i think we shall decide if it is in properties or in processinfo, 

i don't love the idea of looking in both places. What would happen if the decision is contradictory?

**Comment by philbucher** [comment]
How this is selected is clearly documented in the comments in the functions
Also I added tests where I defined contradictory settings to make sure it is not changed in the future.

The reason it is in both places is the following:
Usually we define everything through Properties. The explicit solver however requires the lumped mass matrix to work, which it tells the elements through setting it in the ProcessInfo (globally for all elements). Now if some elements have the same setting in the Properties we still have to compute the lumped mass matrix, hence giving the setting defined in the ProcessInfo priority

**Pull request merged**